docs/guides/unity-catalog.md

```diff
@@ -36,47 +156,132 @@ data "aws_iam_policy_document" "passrole_for_uc" {
 
-resource "aws_iam_role" "metastore_data_access" {
-  name               = "${local.prefix}-uc-access"
-  assume_role_policy = data.aws_iam_policy_document.passrole_for_uc.json
-  tags               = local.tags
-  inline_policy {
-    name = "${aws_s3_bucket.metastore.id}-access"
-    policy = jsonencode({
-      Version = "2012-10-17"
-      Statement = [{
-        Action = [
-          "s3:PutObjectAcl",
+resource "aws_iam_policy" "unity_metastore" {
+  # Terraform's "jsonencode" function converts a
+  # Terraform expression's result to valid JSON syntax.
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Id      = "${local.prefix}-databricks-unity-metastore"
+    Statement = [
+      {
+        "Action": [
+          "s3:GetObject",
+          "s3:GetObjectVersion",
           "s3:PutObject",
+          "s3:PutObjectAcl",
+          "s3:DeleteObject",
           "s3:ListBucket",
-          "s3:GetObjectVersion",
+          "s3:GetBucketLocation"
+        ],
+        "Resource": [
+          aws_s3_bucket.unity_metastore.arn,
+          "${aws_s3_bucket.unity_metastore.arn}/*"
+        ],
+        "Effect": "Allow"
+      }
+    ]
+  })
+  tags = merge(local.tags, {
+    Name = "${local.prefix}-unity-catalog IAM policy"
+  })
+}
+
+resource "aws_iam_policy" "sample_data" {
+  # Terraform's "jsonencode" function converts a
+  # Terraform expression's result to valid JSON syntax.
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Id      = "${local.prefix}-databricks-sample-data"
+    Statement = [
+      {
+        "Action": [
           "s3:GetObject",
-          "s3:GetBucketLocation",
-          "s3:DeleteObject",
-        ]
-        Effect = "Allow"
-        Resource = [
-          aws_s3_bucket.metastore.arn,
-          "${aws_s3_bucket.metastore.arn}/*"
-        ]
-      }]
-    })
-  }
+          "s3:GetObjectVersion",
+          "s3:ListBucket",
+          "s3:GetBucketLocation"
+        ],
+        "Resource": [
+          "arn:aws:s3:::databricks-datasets-oregon/*",
+          "arn:aws:s3:::databricks-datasets-oregon"
+
+        ],
+        "Effect": "Allow"
+      }
+    ]
+  })
+  tags = merge(local.tags, {
+    Name = "${local.prefix}-unity-catalog IAM policy"
+  })
 }
 
+resource "aws_iam_role" "metastore_data_access" {
+  name               = "${local.prefix}-uc-access"
+  assume_role_policy = data.aws_iam_policy_document.passrole_for_uc.json
+  managed_policy_arns = [aws_iam_policy.unity_metastore.arn, aws_iam_policy.sample_data.arn]
+  tags = merge(local.tags, {
+    Name = "${local.prefix}-unity-catalog IAM role"
+  })  
+}
+```
+
+## Create users and groups
+
+A Unity Catalog metastore can be shared across multiple Databricks workspaces. To enable this, Databricks must have a consistent view of users and groups across all workspaces, and has introduced features within the account console to manage this. Users and groups that wish to use Unity Catalog must be created as account level identities
+
+---
+**Note**
```

Comment:
change to this syntax `-> **Note** Databricks does not allow a single user ....`

Reply:
done